fourkitchens/sous-drupal-recipe
# Sous paragraphs

### Description
Adds a sous_paragraphs recipe. The recipe pulls together a number of recipes to provide a sous_paragraphs experience. 

**Recipes included:**

- sous_paragraph_types
- sous_content_types
- sous_admin
- sous_base

**Paragraph types included:**

- Image
- Text
- Text with Media
- Video

The sous_content types recipe contains one content type: page. The Page content type contains a number of base fields and a form display that I modelled on Smith. I adopted Smith's use of field_seo_title and set that as a default in Metatags.

In sous_paragraphs I require sous_content_types and I add a content field that will accept any of the paragraph types. The result has three tabs and looks like this:

<img width="1263" alt="sous-paragraphs-tab1" src="https://github.com/fourkitchens/sous-drupal-recipe/assets/1652730/d39e920b-8b48-4d97-a77e-5c38820fce9e">

<img width="928" alt="sous-paragraphs-tab2" src="https://github.com/fourkitchens/sous-drupal-recipe/assets/1652730/3cab6a2d-f18d-4266-a1ca-1dd7b4ad9316">

<img width="953" alt="sous-paragraphs-tab3" src="https://github.com/fourkitchens/sous-drupal-recipe/assets/1652730/f8c9c9bc-afb2-4bf2-87a7-2471f4078888">


### Testing steps
Install using the installation steps from README.
You should now be able to add a page. Verify it looks like the above. 
Is there anything that you think is missing from this recipe or should be changed to provide a default Sous experience please leave suggestions!
